supabase_auth/types.py

```diff
@@ -303,6 +303,22 @@ class SignInWithPhoneAndPasswordCredentials(TypedDict):
 ]
 
 
+class SignInWithIdTokenCredentials(TypedDict):
+    """
+    Provider name or OIDC `iss` value identifying which provider should be used to verify the provided token. Supported names: `google`, `apple`, `azure`, `facebook`, `keycloak` (deprecated).
+    """
+
+    provider: Literal["google", "apple", "azure", "facebook"]
```

Comment:
Could you add "kakao" as well? https://github.com/supabase/auth-js/pull/845#pullrequestreview-2183768506

It's one of the other providers. Docs are here: https://github.com/supabase/auth/pull/1381